index.html

```diff
@@ -19,25 +19,25 @@ <h1>Daniel Eicher</h1>
     </nav>
   </header>
   <figure>
-    <img src="https://source.unsplash.com/random/600x300" alt="">
+    <img src="https://source.unsplash.com/random/600x300" alt="" width="600" height="300">
     <figcaption>Here will be be a picture of myself</figcaption>
   </figure>
   <section id="interests">
     <h2>Interests</h2>
     <ul class="inline-flex nowrapping">
-      <li>Family</li>
-      <li>Nature</li>
-      <li>Hiking</li>
-      <li>Coding</li>
-      <li>Technology</li>
+      <li>Family,</li>
+      <li>Nature,</li>
+      <li>Hiking,</li>
+      <li>Coding,</li>
+      <li>Technology,</li>
       <li>Reading books</li>
     </ul>
   </section>
-  <section id="contact">
-    <h2>Contact</h2>
-    <ul>
-      <li class="nowrapping"><span class="bold">Phone: </span>+34642104722</li>
-      <li><span class="bold">Mail: </span><a href="mailto:danieleicher@gmx.net">danieleicher@gmx.net</a></li>
+  <section>
+    <h2 id="contact">Contact</h2>
+    <ul class="inline-flex">
+      <li><span class="bold">Phone: </span><br>+34642104722</li>
```

Comment:
for better readability you could divide the number into blocks of 3. for example +34 642 047 22

Reply:
Yes i changed it now, thank you